### PR TITLE
Issue #2219: Provide a way to pass connection properties in DefaultClusteringService

### PR DESCRIPTION
**Pull request by saurabhagas**
Provide a way to pass connection properties in DefaultClusteringService.

### CODE REVIEW

**Comment by terracotta-org** [comment]
Can one of the admins verify this patch?

File: clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java [hunk 1]
```diff
@@ -90,6 +91,7 @@
   DefaultClusteringService(final ClusteringServiceConfiguration configuration) {
     this.configuration = configuration;
     URI ehcacheUri = configuration.getClusterUri();
+    this.properties = configuration.getProperties();
```
Comment (henri-tremblay) [inline clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java]: My problem here is that the timeout is coming from another place (that will override this). I don't like properties coming from two places. I don't have the best answer but I don't think it's a good idea.

**Comment by saurabhagas** [comment]
test this please

**Comment by AbfrmBlr** [comment]
test this please

**Comment by henri-tremblay** [comment]
please rebase

**Comment by saurabhagas** [comment]
test this please

**Pull request merged**